napari/_qt/utils.py

```diff
@@ -123,3 +123,31 @@ def qt_signals_blocked(obj):
     obj.blockSignals(True)
     yield
     obj.blockSignals(False)
+
+
+def connect_model_to_rangeslider(layer, attr, slider, initialize=False):
```

Comment:
I'm not sure I love this function. I guess you want to reuse it more in #837 which is why you've added it here. I'm curious what @jni thinks?

On the one hand it reduces code duplication, but on the other it is another function to keep track of that doesn't do too much.

Reply:
We also don't us this paradigm (i.e functions like this) with our other Qt elements, say the normal slider - I'm curious though if you think we should? In which case I'd be more in favor of this, I'm just not sure it warrants a different approach. That being said I'm very open to keeping it if you think it makes things simpler, and maybe having more convenience functions like this for our other Qt elements would be good too.

Reply:
I originally that because the connection process was more laborious, and it would have made reusing the connections easier (I did want to reuse in #837), but actually now with this refactor, it reduces the connection code enough that i think it's unnecessary.  I can remove it.